ShareBook/Sharebook.Jobs/Executor/JobExecutor.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using ShareBook.Domain;
+using ShareBook.Domain.Common;
+using ShareBook.Repository;
+using System.Diagnostics;
+
+namespace Sharebook.Jobs
+{
+    // inspirado no design pattern chain of responsability
+    // https://pt.wikipedia.org/wiki/Chain_of_Responsibility
+
+    public class JobExecutor : IJobExecutor
```

Comment:
Fiquei com a sensação de que um refactor nesta classe seja necessário. Incomoda um pouco o nível de encadeamento com os condicionais e os vários pontos com "frases explícitas" pelo corpo. Pensando em uma abordagem i18n, talvez seja legal separar essas paradas.

Reply:
ah para de ser mendigo mano. eh só um for e um if. nem eh tão complexo assim kkk.

i18n por enquanto não eh uma prioridade. precisamos conquistar a cidade de são paulo, o estado e o Brasil primeiro. :-D